### PR TITLE
Add sortable document number field to ra second factor overview.

### PR DESCRIPTION
**Pull request by relaxnow**

### CODE REVIEW

File: src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig [hunk 1]
```diff
@@ -28,6 +29,7 @@
                             <td>{{ ('ra.second_factor.search.type.'~secondFactor.type)|trans }}</td>
                             <td>{{ secondFactor.name }}</td>
                             <td>{{ secondFactor.email }}</td>
+                            <td>{{ secondFactor.documentNumber }}</td>
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig]: I think we've used `&mdash` throughout the application when values are missing.

File: src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig [hunk 2]
```diff
@@ -38,6 +40,7 @@
                                         data-sftype="{{ secondFactor.type }}"
                                         data-sfidentityid="{{ secondFactor.identityId }}"
                                         data-sfname="{{ secondFactor.name }}"
+                                        data-sfdocumentnumber="{{ secondFactor.documentNumber }}"
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig]: These data attributes are used for displaying a token revocation confirmation modal. If this is your intention, please [add it to the JavaScript](https://github.com/SURFnet/Stepup-RA/blob/02038b99b6406faea1b1b5f318349f8f2eedf870/src/Surfnet/StepupRa/RaBundle/Resources/public/js/stepup-ra.js#L23-L36). If not, there's no need to add this.

File: src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig (hunk 1 above)
Comment (relaxnow) [inline src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig]: Sounds like a good plan. Fixed.

File: src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig (hunk 2 above)
Comment (relaxnow) [inline src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig]: Okay, removed, we won't need it in revocation.

**Comment by relaxnow** [comment]
@rjkip ready for inspection

**Comment by rjkip** [comment]
Inspection complete. Proceed. :computer:

**Comment by relaxnow** [comment]
Affirmative

**Pull request merged**